.github/workflows/scrape.yml

```diff
@@ -0,0 +1,47 @@
+name: ⚙️ Scrape
+
+on:
+  workflow_call:
+    inputs:
+      object_type:
+        description: What objects to scrape (people, events, bills)?
+        type: string
```

Comment:
You can restrict the choices using a choice type too:

```yml
description: What objects to scrape (people, events, bills)?
  type: choice
  options:
    - people
    - events
    - bills
```

Reply:
I'll keep this in mind for a non-reusable workflow! (I guess you can't set this for reusable workflows, since they're programmatically dispatched.)